pytest/tests/sandbox/fast_forward.py

```diff
@@ -5,23 +5,62 @@
 import sys, time
 import pathlib
 
+from datetime import datetime as dt, timedelta
```

Comment:
Please don’t rename to `dt`. I consider importing symbols from modules bad in and of itself, renaming things makes it even worse.

Reply:
hmm, I can switch it over to the more verbose form, otherwise `datetime import datetime` will be in conflict with `import datetime`

Reply:
Please do that.